sbin/prepareWorkspace.sh

```diff
@@ -77,8 +77,8 @@ checkoutAndCloneOpenJDKGitRepo() {
       exit 1
     fi
   elif [ ! -d "${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}/.git" ]; then
```

Comment:
I know it's not part of this PR but why is this an `elif` instead of just an `else` given that the condition is the same other than the `!`?

Reply:
I suspect there's some history, possibly this PR https://github.com/AdoptOpenJDK/openjdk-build/commit/b60282a390d0fc0685fef376603716ab84a4ee63
whereby the first if was different before